src/LiveComponent/src/EventListener/LiveComponentSubscriber.php

```diff
@@ -141,11 +142,19 @@ public function onKernelController(ControllerEvent $event): void
 
         $this->container->get(LiveComponentHydrator::class)->hydrate($component, $data);
 
-        // extra variables to be made available to the controller
-        // (for "actions" only)
-        parse_str($request->query->get('values'), $values);
-        $request->attributes->add($values);
         $request->attributes->set('_component', $component);
+
+        if (\is_string($queryString = $request->query->get('args'))) {
```

Comment:
This was needed to resolve deprecation: `10x: parse_str(): Passing null to parameter #1 ($string) of type string is deprecated`. Also it would fail with other types, so added sanity check

Reply:
Were we going to call `args` `_live_args`?